docs/samples/canary-testing/README.md

```diff
@@ -0,0 +1,151 @@
+# Canary testing
+
+This tutorial demonstrates canary testing for KServe inference services using model metrics written to Prometheus. We will use [Iter8](https://iter8.tools) to read the metrics from Prometheus and validate service-level objectives (SLOs) for two model versions. [Iter8](https://iter8.tools) is an open-source Kubernetes release optimizer that makes it easy to ensure that your ML models perform well and maximize business value.
+
+![Canary testing](canary.png)
+
+***
+
+> Performance (load) testing of KServe inference services is described [here](../performance-testing/README.md). This tutorial focuses on canary testing using Prometheus metrics. The main steps in this tutorial are:
+> 1. [Deploy an InferenceService](#deploy-an-inferenceservice)
+> 2. [Deploy canary version](#deploy-canary-version)
+> 3. [Launch an Iter8 experiment](#launch-an-iter8-experiment)
+> 4. [View experiment report](#view-experiment-report)
+
+## Install Prometheus
+
+Install Prometheus monitoring for KServe [using these instructions](https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring#install-prometheus).
+
+## Deploy an InferenceService
+
+Create an InferenceService.
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model"
+EOF
+```
+
+## Deploy canary version
+
+Update the inference service with a canary model, configured to receive 10% of prediction requests.
+
+```shell
+kubectl apply -f - <<EOF
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-iris"
+spec:
+  predictor:
+    canaryTrafficPercent: 10
+    model:
+      modelFormat:
+        name: sklearn
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model-2"
+EOF
+```
+
+Verify that your inference service is ready.
+
+```shell
+kubectl wait --for=condition=Ready --timeout=600s isvc/sklearn-iris
+```
+
+## Install Iter8 CLI
```

Comment:
So from what i read from the tutorial, iter8 is now a client tool and no longer requires installing manifest in kubernetes, right?